ElementX/Sources/Screens/HomeScreen/HomeScreenModels.swift

```diff
@@ -175,3 +178,88 @@ struct HomeScreenRoom: Identifiable, Equatable {
                        isPlaceholder: true)
     }
 }
+
+enum RoomListFilter: Int, CaseIterable, Identifiable {
+    var id: Int {
+        rawValue
+    }
+    
+    case people
+    case rooms
+    case unreads
+    case favourites
+    case lowPriority
+    
+    var localisedName: String {
```

Comment:
As much as I prefer the UK spelling, we should be using US in code.
```suggestion
    var localizedName: String {
```